_layouts/navbar_logged_out.jade

```diff
@@ -2,10 +2,10 @@ include ../_mixins/links.jade
 
 .fixed.landing-page
 
-    if (custom_navigation)
+    if (global.custom.custom_navigation)
```

Comment:
since we can't use ```global.custom?.custom_navigation``` here can you add the extra check for global.custom: ```if (global.custom && global.custom.custom_navigation)